cfn-resources/autogen/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: submit test
+tags=logging callback metrics scheduler
+cgo=0
+goos=linux
+goarch=amd64
+#export TAGS
+#export CGO_ENABLED
+#export GOARCH
+# Set next value at compile time for fallback log-level
+ldXflags=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info
+ldXflagsD=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug
+
+# example for resource: MongoDB::Atlas::Teams
+# example for go path: github.com/mongodb/mongodbatlas-cloudformation-resources/invites
+# example for go path: github.com/mongodb/mongodbatlas-cloudformation-resources/indexes
+
+create:
+	cd .. && \
+	mkdir $(dirName) && cp autogen/configs/mongodb-atlas-$(resourceName).json $(dirName)/mongodb-atlas-$(resourceName).json && \
```

Comment:
this resourceName we should turn it into lower case, im sorry i cant provide any code for this, im kinda ignorant about this type of code, jajaja but can we do something like this? 

"$a" | tr '[:upper:]' '[:lower:]'

because is creating it as mongodb-atlas-CustomDBRole.json, and it should be mongodb-atlas-customdbrole.json